presto-main/src/main/java/com/facebook/presto/operator/scalar/ThetaSketchFunctions.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.scalar;
+
+import com.facebook.presto.common.block.Block;
+import com.facebook.presto.common.block.BlockBuilder;
+import com.facebook.presto.common.type.RowType;
+import com.facebook.presto.spi.function.Description;
+import com.facebook.presto.spi.function.ScalarFunction;
+import com.facebook.presto.spi.function.SqlType;
+import com.google.common.collect.ImmutableList;
+import io.airlift.slice.Slice;
+import org.apache.datasketches.memory.Memory;
+import org.apache.datasketches.theta.CompactSketch;
+
+import java.util.Optional;
+
+import static com.facebook.presto.common.type.DoubleType.DOUBLE;
+import static com.facebook.presto.common.type.IntegerType.INTEGER;
+
+public class ThetaSketchFunctions
+{
+    private ThetaSketchFunctions()
+    {
+    }
+
+    @ScalarFunction(value = "sketch_theta_estimate")
+    @Description("Get the estimate of unique values from a theta sketch")
+    @SqlType("double")
+    public static double thetaSketchEstimate(@SqlType("varbinary") Slice input)
+    {
+        CompactSketch sketch = CompactSketch.wrap(Memory.wrap(input.getBytes()));
+        return sketch.getEstimate();
+    }
+
+    private static final RowType SUMMARY_TYPE = RowType.from(ImmutableList.of(
+                new RowType.Field(Optional.of("estimate"), DOUBLE),
+                new RowType.Field(Optional.of("theta"), DOUBLE),
+                new RowType.Field(Optional.of("upper_bound_std1"), DOUBLE),
```

Comment:
why _std1?

Reply:
I did `std1` to indicate a single standard deviation since the sketch API allows you to query for any N standard deviations. I suppose if we stick with a single standard deviation I can just do `upper_bound_std` and `lower_bound_std`

Reply:
The most recent commit changes this to `upper_bound_std` and `lower_bound_std`